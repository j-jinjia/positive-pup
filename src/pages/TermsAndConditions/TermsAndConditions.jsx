import Layout from "../../components/Layout/Layout";

const TermsAndConditions = () => {
  return (
  <Layout>
    <h1>Terms and conditions</h1>
  </Layout>
  )
};

export default TermsAndConditions;
