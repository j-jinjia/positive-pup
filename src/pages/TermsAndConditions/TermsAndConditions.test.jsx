import { customRender } from "../../utils/testUtils";
import TermsAndConditions from "./TermsAndConditions";

it("should render the terms and conditions page", () => {
  const { container } = customRender(<TermsAndConditions />);

  expect(container).toMatchSnapshot();
});