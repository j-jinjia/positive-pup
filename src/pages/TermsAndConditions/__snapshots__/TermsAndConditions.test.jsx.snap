// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the terms and conditions page 1`] = `
<div>
  <div
    class="layout"
  >
    <h1>
      Terms and conditions
    </h1>
    <div
      class="footer"
    >
      <div
        class="footer__image-container"
      >
        <img
          alt="logo"
          class="footer__logo"
          src="footer-logo.svg"
        />
      </div>
      <ul
        class="footer__links-container"
      >
        <li
          class="footer__list-item"
        >
          <a
            class="footer__link"
            href="/about"
          >
            About
          </a>
        </li>
        <li
          class="footer__list-item"
        >
          <a
            class="footer__link"
            href="/services"
          >
            Services
          </a>
        </li>
        <li
          class="footer__list-item"
        >
          <a
            class="footer__link"
            href="/gallery"
          >
            Gallery
          </a>
        </li>
        <li
          class="footer__list-item footer__list-item--desktop"
        >
          <a
            class="footer__link"
            href="/terms-and-conditions"
          >
            Terms & Conditions
          </a>
        </li>
        <li
          class="footer__list-item footer__list-item--mobile"
        >
          <a
            class="footer__link"
            href="/terms-and-conditions"
          >
            T&C's
          </a>
        </li>
        <li
          class="footer__list-item"
        >
          <a
            class="footer__link"
            href="/faqs"
          >
            FAQ's
          </a>
        </li>
        <li
          class="footer__list-item"
        >
          <a
            class="footer__link"
            href="/get-in-touch"
          >
            Contact
          </a>
        </li>
      </ul>
      <hr
        class="footer__line-break"
      />
      <div
        class="footer__secondary-links-container"
      >
        <div
          class="footer__copyright-text"
        >
          Â© 2022 The Positive Pup. All rights reserved.
        </div>
        <div
          class="footer__social-links"
        >
          <a
            href="https://www.facebook.com/thepositivepupdogtraining"
            target="blank"
          >
            <img
              alt="link to facebook"
              src="facebook-logo.svg"
            />
          </a>
          <a
            href="https://www.instagram.com/the_positivepup/"
            target="blank"
          >
            <img
              alt="link to instagram"
              src="instagram-logo.svg"
            />
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;
